packages/codemod/src/utils/prop-utils.ts

```diff
@@ -173,6 +173,25 @@ export function updatePropValues(
   });
 }
 
+export function updateStaticPropKeys(
```

Comment:
It sound like this function should rely on the util of `updatePropValues` or `setPropValue`

Reply:
we talked about it, understood why it can't be used